### PR TITLE
Added 2FA instructions to TFE section

### PR DESCRIPTION
**Pull request by rkst**
• Adds navigation link under Terraform Enterprise -> Users, Teams, Organizations
• First pass at the 2FA docs and screenshots

### CODE REVIEW

File: content/source/docs/enterprise/users-teams-organizations/2fa.html.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+After successful set-up you will also have the ability to view backup codes, as well as disable two-factor authentication.
+
+![The two-factor authentication page after successful set-up](./images/2fa-backup-codes.png)
+
+## Logging in with Two-factor authentication
+
+After two-factor authentication has been successfully set-up you will need to enter the code from your TOTP-compliant application or from an SMS sent to your approved SMS-enabled phone number on login.
+
+If necessary you can also use a backup code by clicking "Use a recovery code". Please remember that each backup code can only be used to log in once.
```
Comment (rkst) [inline content/source/docs/enterprise/users-teams-organizations/2fa.html.md]: We call them "backup codes" in the user settings menu but on the login page we refer to them as "recovery codes". It's a minor thing but I can file it in TFE Feedback if people agree this is not an intended inconsistency.

**Comment by nfagerlund** [comment]
@rkst I do! And... during review I forgot that I usually do that 😅

I usually put it in a callout (start a line with our site-specific `->` syntax marker) that starts with "`**API:**`"

For example, https://terraform.io/docs/enterprise/workspaces/creating.html

**Comment by rkst** [comment]
@nfagerlund I added it under the sub-heading for Setting up instead of putting it at the top because I couldn't find and API endpoint for managing it an an Org level. Feel free to move it to the top if you'd prefer it there.

**Comment by nfagerlund** [comment]
I found the API for managing the org policy, so I'll add that separately to the other header.

**Pull request merged**